.travis.yml

```diff
@@ -25,13 +25,13 @@ stage: test
 env:
   global:
   - COVERAGE_OPTION='./node_modules/.bin/nyc'
-  - NODE_VERSION=6.11.4
+  - NODE_VERSION=8.9.3
   matrix:
   - MONGODB_VERSION=3.2.13
   - MONGODB_VERSION=3.4.4
   - PARSE_SERVER_TEST_DB=postgres
   - PARSE_SERVER_TEST_CACHE=redis
-  - NODE_VERSION=8.7
+  - NODE_VERSION=9.3.0
```

Comment:
All Node 9.x versions thus far are a complete unusable crap, because their ditribute with the version of NPM that doesn't work right with Node 9.x

Reply:
Seems to be fine, as the tests are passing. The goal of that line is to make sure it’s all forward compatible